qlik-core/engine-deployment.yaml

```diff
@@ -1,11 +1,14 @@
-apiVersion: extensions/v1beta1
+apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: engine
   labels:
     app: engine
 spec:
   replicas: 2
+  selector:
```

Comment:
What does this add?

Reply:
Introduced as a requirement in apiversion apps/v1 , read more about the selectors here: 
https://medium.com/@zwhitchcox/matchlabels-labels-and-selectors-explained-in-detail-for-beginners-d421bdd05362